goentri.com.pitchlane_subdomains.json

```diff
@@ -0,0 +1,21 @@
+{
+    "providerId": "goentri.com",
+    "providerName": "Entri",
+    "serviceId": "pitchlane_subdomains",
+    "serviceName": "PitchlaneSubdomains",
+    "version": 1,
+    "logoUrl": "https://cdn.goentri.com/logo.svg",
+    "description": "Allows user to easily set up domain using Entri",
+    "syncPubKeyDomain": "goentri.com",
+    "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com, app.goentri.com",
+    "variableDescription":"cnameHost is the subdomain name and cnamePointsto is to where the subdodoain is pointed",
+    "sharedProviderName": true,
+    "records": [
+      {
+        "type":"CNAME",
+        "host":"%cnameHost%",
```

Comment:
Thank you so much for clarifying that Pawel, I don't think they are going to use this template in different use cases, however, I will make the changes to avoid this behavior anyway.

Reply:
@pawel-kow Looks like the requested change was completed. Can we proceed with the merge? Apologies, if I am missing anything else. Thanks...

Reply:
Hi @pawel-kow, do you need any more updates to merge this template?